docs/extend/extend-apiml/zaas-client.md

```diff
@@ -88,6 +89,21 @@ cookie or in an Authorization header. You then receive the `ZaasToken` Object in
 ZaasToken query(HttpServletRequest request) throws ZaasClientException;
 ```
 
+### Validate the OIDC token (`validateOidc`)
```

Comment:
The only mention I found of the `/oidc-token/validate` endpoint is here: https://docs.zowe.org/v2.18.x/extend/extend-apiml/api-mediation-oidc-authentication/#api-ml-fails-to-validate-the-oidc-access-token-with-the-distributed-identity-provider

Can you update the return code to 204 there?

Reply:
it should go to master / 2.18 as well I think

Reply:
I meant the updated return code